tests/requirements-libraries.txt

```diff
@@ -42,7 +42,7 @@ pytz==2021.3
 requests==2.26.0
 # simplejson is used for text_c_extension
 simplejson==3.17.5
-sphinx==2.4.4 # pyup: ignore
+sphinx==4.2.0
```

Comment:
Can we also pin docutils too since it's a rather volatile dependency of sphinx?

Reply:
OK, will do once the current CI run hands over its verdict.

Reply:
Looks like recent versions of `sphinx` properly pin the `docutils` version themselves (`Requires-Dist: docutils (<0.18,>=0.14)`). 

I tried pinning the latest version, but that obviously won't work. So if we decide to pin `docutils` ourselves, we'll need to keep it in sync with `sphinx`' requirements...

Reply:
I think we shouldn't pin docutils - that way, when it breaks, we know.

Reply:
> I think we shouldn't pin docutils - that way, when it breaks, we know.

Well PyUp's pull requests would tell us if a new breaking version appears but if Sphinx is intentionally putting an upper bound on the requirement then pinning docutils ourselves will just lead to PyUp trying to upgrade our docutils version to one which contradicts Sphinx's upper bound so in that case I agree - let's leave docutils unpinned.